Running-Mastodon/Production-guide.md

```diff
@@ -111,7 +111,12 @@ server {
 
 ## Running in production without Docker
 
-It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo -u mastodon`.
+It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo su mastodon`.
+
+This command will create the user as needed:
+
+    sudo adduser --disabled-password --gecos "" mastodon
```

Comment:
Okay I got it fixed! `sudo -Hsu` is needed
![screen shot 2017-06-17 at 2 04 24 pm](https://user-images.githubusercontent.com/495512/27249977-1bf2d22c-5366-11e7-89ea-25a9dc01ceef.png)